summingbird-storm/src/main/scala/com/twitter/summingbird/storm/builder/EdgeType.scala

```diff
@@ -56,7 +54,7 @@ object EdgeType {
 }
 
 private object EdgeTypeInjections {
-  def forItem[T]: Injection[T, JList[AnyRef]] = new Injection[T, JList[AnyRef]] {
+  case class ForItem[T]() extends Injection[T, JList[AnyRef]] {
```

Comment:
Fields is redundant actually since that's almost implied because of it being on EdgeType. I think we can just remove For and call it Item.